client/i18n/locales/english/intro.json

```
@@ -805,6 +798,45 @@
       }
     }
   },
+  "project-euler": {
+    "title": "Project Euler",
+    "intro": [
+      "Complete the programming challenges below, from the massive Project Euler archives. These will harden your algorithm and mathematics knowledge.",
+      "These problems range in difficulty and, for many, the experience is inductive chain learning. That is, by solving one problem, it will expose you to a new concept that allows you to undertake a previously inaccessible problem. Can you finish them all?"
+    ],
+    "blocks": {
+      "project-euler-problem-1-100": {
+        "title": "Project Euler Problem 1 to 100",
```

This will require more changes, but I think it would be better to go with the plural "problems" for these blocks:

```suggestion
        "title": "Project Euler Problems 1 to 100",
```